src/screens/TopicPageScreen/MemoizedListComponent.js

```diff
@@ -3,6 +3,7 @@ import React from 'react';
 import SimpleToast from 'react-native-simple-toast';
 import {Dimensions, StatusBar, StyleSheet, View} from 'react-native';
 
+import {useRoute} from '@react-navigation/core';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `useRoute` from `@react-navigation/core` is added. This hook allows access to the route prop in any component. It's a good practice to use this hook when you need to access parameters or the name of the current route.
```diff
+ import {useRoute} from '@react-navigation/core';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->